var/spack/repos/builtin/packages/py-pyonmttok/package.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPyonmttok(CMakePackage, PythonExtension):
+    """Tokenizer is a fast, generic, and customizable text tokenization library for
+    C++ and Python with minimal dependencies."""
+
+    homepage = "https://github.com/OpenNMT/Tokenizer/"
+    git = "https://github.com/OpenNMT/Tokenizer.git"
+
+    maintainers("meyersbs", "aweits")
+
+    version("1.37.1", tag="v1.37.1", submodules=True)
+
+    extends("python")
+
+    # Uses setuptools.installer, deprecated in:
+    # https://setuptools.pypa.io/en/stable/history.html#v59-1-0
+    depends_on("py-setuptools@:59.0.1", type="build")
+    depends_on("py-pybind11", type="build")
+    depends_on("py-pip@19.3:", type="build")
+    depends_on("python@3.6:3.11", type=("build", "run"))
+    depends_on("cmake@3.1.0:", type="build")
+
+    variant("cli", default=True, description="Build CLI")
+
+    def setup_build_environment(self, env):
+        env.append_path("SPACK_INCLUDE_DIRS", prefix.include)
+        env.append_path("SPACK_LINK_DIRS", prefix.lib)
+        env.append_path("SPACK_LINK_DIRS", prefix.lib64)
```

Comment:
Is this necessary?

Reply:
Same thing here; python bits wouldn't install

Reply:
Can you share the error message?

Reply:
The python bits wouldn't install without these lines; it would build the cmake bits just fine, but then fail during the install_python phase:

```
737      Running setup.py install for ctranslate2: started
738      Running command Running setup.py install for ctranslate2
739      running install
...
742      running build
743      running build_py
744      creating build
745      creating build/lib.linux-x86_64-3.10
746      creating build/lib.linux-x86_64-3.10/ctranslate2
...
>> 786      /usr/bin/ld: cannot find -lctranslate2
>> 787      collect2: error: ld returned 1 exit status
788      error: command '/home/bsmits/spack/lib/spack/env/gcc/g++' failed with exit code 1
789      error: subprocess-exited-with-error
790    
791      × Running setup.py install for ctranslate2 did not run successfully.
792      │ exit code: 1
793      ╰─> See above for output.
```